### PR TITLE
.maintain/monitoring: Add alerting rule tests

### PR DESCRIPTION
**Pull request by mxinden**
Adding a basic set of Prometheus alerting rule tests adapted from https://github.com/w3f/polkadot-charts/blob/master/tests/prometheus/polkadot/alertrules.yaml.

~~I will follow up with another pull request to have these checked via Promtool on our CI.~~ (Done)

//CC @fgimenez

### CODE REVIEW

**Comment by tomaka** [comment]
Ok I didn't realize these were only tests for existing rules.

**Comment by mxinden** [comment]
> Ok I didn't realize these were only tests for existing rules.

I am not quite sure I follow. How should we have tests for non-existing rules @tomaka?

**Comment by tomaka** [comment]
When I opened this PR I thought that it was modifying the rules themselves.

**Comment by mxinden** [comment]
@ddorgan @gabreal could you give this pull request a review?

File: .gitlab-ci.yml [hunk 1]
```diff
@@ -367,6 +367,7 @@ test-prometheus-alerting-rules:
     - curl -L https://github.com/prometheus/prometheus/releases/download/v2.19.0/prometheus-2.19.0.linux-amd64.tar.gz --output prometheus.tar.gz
     - tar -xzf prometheus.tar.gz
     - ./prometheus-*/promtool check rules .maintain/monitoring/alerting-rules/alerting-rules.yaml
```
Comment (gabreal) [inline .gitlab-ci.yml:369]: I created a pr for adding promtool directly to the tools image: https://github.com/paritytech/scripts/pull/209

File: .maintain/monitoring/alerting-rules/alerting-rules.yaml [hunk 2]
```diff
@@ -12,22 +12,25 @@ groups:
...
 
   ##############################################################################
   # Block production
   ##############################################################################
 
   - alert: LowNumberOfNewBlocks
     annotations:
-      message: 'Less than one new block per minute on instance {{ $labels.instance }}.'
+      message: 'Less than one new block per minute on instance {{
+      $labels.instance }}.'
```
Comment (gabreal) [inline .maintain/monitoring/alerting-rules/alerting-rules.yaml:25]: imo this kind of line break makes the file less readable

**Pull request merged**